ar_SA.json

```diff
@@ -1552,6 +1554,8 @@
 	"property.origin.name.on": "على",
 	"property.origin.name.placeholder": "Enter a name for your origin.",
```

Comment:
Does property.origin.name.placeholder ever get shown to the user?

Reply:
Yes, it shows in origin details
![image](https://github.com/mileweb/ngportal-language/assets/65930352/b7daaf0c-963d-4238-9197-a0c4d9b999b0)

Reply:
If it is shown, why isn't it also translated ?

Reply:
As we already know, if you look at the files, most of them are in English and we started to translate them at some point. Currently, we translate into all supported languages, but many previous parts are still supported in English.